ivy/utils/backend/sub_backend_handler.py

```diff
@@ -0,0 +1,148 @@
+import os
+import importlib
+from types import ModuleType, FunctionType
+
+import ivy
+from ivy.func_wrapper import _wrap_function
+from ivy.utils.exceptions import IvyException
+
+_sub_backend_dict = dict()
+_backend_to_sub_backends_dict = dict()
+# torch sub_backends
+
+_sub_backend_dict["xformers"] = "ivy.functional.backends.torch.sub_backends.xformers"
```

Comment:
Also similiar to the other note on the init file, this should be done dynamically.